modules/distr-tracing-config-otel-send-data-to-monitoring-stack.adoc

```diff
@@ -0,0 +1,37 @@
+////
+This module included in the following assemblies:
+-distr_tracing_install/distributed-tracing-deploying-otel.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-config-otel-collector_monitoring_{context}"]
+= Sending metrics to the monitoring stack
+
+.Sample `+PodMonitor+` that configures monitoring stack to scrape OpenTelemetry collector and removes duplicated labels.
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: otel-collector
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: otel-collector
+  podMetricsEndpoints:
+  - port: metrics <1>
+  - port: promexporter <2>
+    relabelings:
+    - action: labeldrop
+      regex: pod
+    - action: labeldrop
+      regex: container
+    - action: labeldrop
+      regex: endpoint
+    metricRelabelings:
+    - action: labeldrop
+      regex: instance
+    - action: labeldrop
+      regex: job
+----
+<1> Collector's internal metrics port
+<2> Collector's Prometheus exporter port
```

Comment:
```suggestion
<1> The OpenTelemetry collector internal metrics port
<2> The OpenTelemetry collector Prometheus exporter port
```
explain properly what these are and maybe what the possible values are or should be - if these values aren't literal use something like `<port-name>` to show where the user's value would go

Reply:
+1 good comment. I have applied your change and added more explanation.